Isosurfacing_3/include/CGAL/Isosurfacing_3/edge_intersection_oracles_3.h

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2022-2024 INRIA Sophia-Antipolis (France), GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Julian Stahl
+//                 Mael Rouxel-Labbé
+
+#ifndef CGAL_ISOSURFACING_3_EDGE_INTERSECTION_ORACLES_3_H
+#define CGAL_ISOSURFACING_3_EDGE_INTERSECTION_ORACLES_3_H
+
+#include <CGAL/license/Isosurfacing_3.h>
+
+#include <CGAL/assertions.h>
+
+namespace CGAL {
+namespace Isosurfacing {
+
+/**
+ * \ingroup IS_Domain_helpers_grp
+ *
+ * \cgalModels{IsosurfacingEdgeIntersectionOracle_3}
+ *
+ * \brief The class `Dichotomy_edge_intersection` uses a dichotomy to find the intersection point
+ * between an edge and the isosurface.
+ *
+ * This class is for example suitable to be used as the `EdgeIntersectionOracle` template parameter of isosurfacing
+ * domain classes when values are computed using an implicit function.
+ * It is however not optimal when the values are interpolated from discrete values
+ * since the intersection can be computed analytically in this case.
+ *
+ * \sa `CGAL::Isosurfacing::Linear_interpolation_edge_intersection`
+ * \sa `CGAL::Isosurfacing::Marching_cubes_domain_3`
+ * \sa `CGAL::Isosurfacing::Dual_contouring_domain_3`
+ * \sa `CGAL::Isosurfacing::Value_field_3`
```

Comment:
There is no link for `Value_field_3` in [doc v3](https://cgal.geometryfactory.com/~mrouxell/Isosurfacing_3/V3/Isosurfacing_3/structCGAL_1_1Isosurfacing_1_1Dichotomy__edge__intersection.html)